images/metering-ansible-operator/roles/meteringconfig/tasks/configure_metastore.yml

```diff
@@ -0,0 +1,9 @@
+- name: Get metastore secret
+  block:
+    - name: Get Metastore storage credentials secret
+      k8s_info:
+        api_version: v1
+        kind: Secret
+        name: "{{ meteringconfig_spec.hive.spec.config.db.secretName | b64decode }}"
+        namespace: "{{ meta.namespace }}"
+      register: metastore_secret
```

Comment:
We need to add a conditional that this block only gets run when the `meteringconfig_spec.hive.spec.config.db` configuration has been specified in the MeteringConfig CR and `secretName` is not an empty field.